### ISSUE TITLE
Determine desired look and feel of site

### ISSUE THREAD

**Issue by lanesawyer** [lanesawyer/gshe#4]

**Comment by Stackranker** [comment]
Remind me- will the product be available on all devices (e.g. PC, Mac, iPhone, iPad, Android)? I'm considering delivering a live demo on 3 April if the timing works out, and I'm wondering what options to consider in terms of device selection.

**Comment by lanesawyer** [comment]
Yes, the site will work on all devices. I'm building the site to be responsive so it'll work on any size screen.
